### DISCUSSION TITLE
What's the problem with 24.03 for StarBook MkVI-Intel?

### DISCUSSION THREAD

**Discussion by sim31** [StarLabsLtd/firmware#173] [General]
[I see this on fwupd website](https://fwupd.org/lvfs/devices/com.starlabs.B6-I.coreboot):
![image](https://github.com/StarLabsLtd/firmware/assets/7878290/6ec3054e-5be9-4033-9387-7b47623859de)

I already had updated to 24.03, wondering if it's anything serious...

**Comment by Sean-StarLabs** [comment]
It's just users trying to install it without installing the required components (*cough arch users*), which counts as a failed install.

Nothing wrong with the firmware itself

**Comment by sim31** [comment]
I'm an arch user but I'm installing from ubuntu lately. I was having [this issue](https://github.com/StarLabsLtd/firmware/issues/83) last time I tried it from arch. What are the current problems with installs from arch? I see that the [flashrom package](https://archlinux.org/packages/extra/x86_64/flashrom/) finally got updated in arch repos.

**Comment by Sean-StarLabs** [comment]
They fail, because of various dependencies issues - like on your other issue, that would have added 4 failed install counts to that firmware.
